src/components/v2/V2Project/V2ProjectToolsDrawer/V2ProjectToolsDrawer.tsx

```diff
@@ -0,0 +1,98 @@
+import { Trans } from '@lingui/macro'
```

Comment:
This file is an exact duplicate of the previous tools draw component, but uses all-v2 hooks/components.